<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Card Game - Poker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gameView">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>571</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1B5E20; border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>921</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="styleSheet">
        <string notr="true">background-color: #4CAF50; color: white; font-weight: bold; border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Start Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="checkHandButton">
       <property name="styleSheet">
        <string notr="true">background-color: #FFEB3B; color: #212121; font-weight: bold; border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Check Hand</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="swapButton">
       <property name="styleSheet">
        <string notr="true">background-color: #FFEB3B; color: #212121; font-weight: bold; border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Swap Cards</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextRoundButton">
       <property name="styleSheet">
        <string notr="true">background-color: #FFC107; color: #212121; font-weight: bold; border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Next Round</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishGameButton">
       <property name="styleSheet">
        <string notr="true">background-color: #F44336; color: white; font-weight: bold; border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Finish Game</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>149</width>
      <height>200</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="computerScoreLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 16px; color: #FFD740; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Computer Score: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="playerScoreLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 16px; color: #FFD740; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>User Score: 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>160</width>
      <height>146</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="RoundLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 16px; color: #FFD740; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Round: 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RemainingCardsLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 16px; color: #FFD740; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Remaining Cards: 52</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="gameView_2">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>10</y>
      <width>561</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1B5E20; border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gameView_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1B5E20; border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gameView_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>211</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1B5E20; border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>261</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="computerRoundWins">
       <property name="styleSheet">
        <string notr="true">font-size: 16px; color: #FFD740; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Computer Round Wins: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="playerRoundWins">
       <property name="styleSheet">
        <string notr="true">font-size: 16px; color: #FFD740; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Player Round Wins: 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="playerHandTypeLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>411</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px; color: #E0F2F1; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Player Hand:</string>
    </property>
   </widget>
   <widget class="QLabel" name="computerHandTypeLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>361</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px; color: #E0F2F1; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Computer Hand:</string>
    </property>
   </widget>
   <zorder>gameView_3</zorder>
   <zorder>gameView_4</zorder>
   <zorder>gameView_2</zorder>
   <zorder>gameView</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>verticalLayoutWidget_3</zorder>
   <zorder>playerHandTypeLabel</zorder>
   <zorder>computerHandTypeLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
